eyepop/eyepopsdk.py

```diff
@@ -71,7 +86,7 @@ def workerEndpoint(
         )
 
         if not is_async:
-            endpoint = SyncWorkerEndpoint(endpoint)
+            return SyncWorkerEndpoint(endpoint)
```

Comment:
for proper type checking: 
```
endpoint = WorkerEndpoint(...
``` 

implicitly sets the type for `endpoint = ` which breaks the SyncWorkerEndpoint type assignment. 

If we try to properly static type the endpoint var: 

```
endpoint: WorkerEndpoint | SyncWorkerEndpoint = WorkerEndpoint(...
``` 

This creates a cascade of type errors. By returning straight away we let `py.typed` def be able to automatically figure things out for users and provide a least a little typechecking for them.